snapcraft/internal/lifecycle.py

```diff
@@ -40,7 +40,8 @@
 _TEMPLATE_YAML = r'''name: # the name of the snap
 version: # the version of the snap
 summary: # 79 char long summary
-description: # A longer description for the snap
+description: # a longer description for the snap
+confinement: devmode # devmode means the snap doesn't support confinement
```

Comment:
after a quick talk, I prefer yours better and love sprints for the speed of sound discussions.